armi/physics/neutronics/globalFlux/globalFluxInterface.py

```diff
@@ -339,6 +339,7 @@ def __init__(self, label: Optional[str] = None):
         # can happen in eig if Fredholm Alternative satisfied
         self.includeFixedSource = False
         self.eigenvalueProblem = True
+        self.copyOutput = True
```

Comment:
Should this be added to armi/physics/neutronics/settings.py? Otherwise to change the setting a user would have to go into the source code.

Reply:
A user would be extending `globalFluxInterfaceUsingExecuters` for their use case, and they can set `copyOutput` when they configure the options for that specific type of executer. The user could add a case setting to control the `copyOutput` setting for their specific executer. That's what I've done with two of our internal repos. This provides finer grain control because you can have several different interfaces run executers during a run, and you might want some to copy their output back but not others. A general case setting that forces all interfaces to adopt the same `copyOutput` value would be less versatile.

Reply:
Ok got it. 

This is likely a separate issue, but are the options in the initializer/constructor of globalFluxInterface.py::GlobalFluxOptions documented anywhere?

Reply:
I think it would be a good idea. But we can do that in a separate issue and PR.